recipe/meta.yaml

```diff
@@ -61,10 +61,12 @@ outputs:
     build:
       merge_build_host: False
       detect_binary_files_with_prefix: False
+      ignore_run_exports:
+        - __glibc
```

Comment:
Why is this ok?

Reply:
The selector at https://github.com/conda-forge/linux-sysroot-feedstock/blob/master/recipe/meta.yaml#L59-L61 is wrong. `x86_64` is true for all builds because the build is noarch and running on linux-64

Reply:
oh frack. 🤦 i'll fix